albumentations/augmentations/transforms.py

```diff
@@ -126,19 +139,39 @@ def update_params(self, params, **kwargs):
         rows = params["rows"]
         cols = params["cols"]
 
-        if rows < self.min_height:
-            h_pad_top = int((self.min_height - rows) / 2.0)
-            h_pad_bottom = self.min_height - rows - h_pad_top
+        if self.min_height is not None:
+            if rows < self.min_height:
+                h_pad_top = int((self.min_height - rows) / 2.0)
+                h_pad_bottom = self.min_height - rows - h_pad_top
+            else:
+                h_pad_top = 0
+                h_pad_bottom = 0
         else:
-            h_pad_top = 0
-            h_pad_bottom = 0
+            if rows > self.pad_height_divisor:
+                pad_rows = rows % self.pad_height_divisor
+                pad_rows = self.pad_height_divisor - pad_rows if pad_rows > 0 else 0
+            else:
+                pad_rows = self.pad_height_divisor - rows
```

Comment:
I think we can simplify that code by remove condition, because first first branch works fine in all situations.
We need this condition if we want to speedup code, but I not sure that we achieve real speedup there.